test/typescript/diff-deploy-destroy/test.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Testing a full cycle of diff, deploy and destroy
+ *
+ * @group typescript
+ */
+
+import { TestDriver } from "../../test-helper";
+
+describe("full integration test", () => {
+  let driver: TestDriver;
+
+  beforeAll(() => {
+    driver = new TestDriver(__dirname)
+    driver.setupTypescriptProject()
+  });
+
+  test("diff", () => {
+    expect(driver.diff()).toMatchInlineSnapshot(`
+      "Stack: hello-deploy
+      Resources
+       + NULL_RESOURCE       test                null_resource.test
```

Comment:
This did lose the test where the feature flags aren't enabled, but that seems fine.

Reply:
Hm, I'll check if that's tested somewhere else.

Reply:
See 0a1ab00